web3/_utils/abi.py

```diff
@@ -456,27 +453,24 @@ def merge_args_and_kwargs(
     duplicate_args = kwarg_names.intersection(args_as_kwargs.keys())
     if duplicate_args:
         raise TypeError(
-            "{fn_name}() got multiple values for argument(s) '{dups}'".format(
-                fn_name=function_abi['name'],
-                dups=', '.join(duplicate_args),
-            )
+            f"{function_abi.get('name')}() got multiple values for "
+            f"argument{'s' if len(duplicate_args) > 1 else ''} "
```

Comment:
I think for better code readability we can do something like:

```
    if duplicate_args:
        s = 's' if len(duplicate_args) > 1 else ''
        duplicate_args = ', '.join(duplicate_args)
        raise TypeError(
            f"{function_abi.get('name')}() got multiple values for argument{s} '{duplicate_args}'"
        )
```

Reply:
Well we talked it through too and static messages (predictable text) is better practice, with only values / variables being dynamic in the message. It can also be a source of bugs we have to go through and fix since it's not tested, etc. This seems like the only change left. I wouldn't mind making the change though if you want and merging it. Let me know!

Reply:
Alright. Well, i don't want to bother you with this issue, but if you have time, you are good to go. At the same time i can do it myself, cause there are other strings i have changed that look like this. And oh, forgot to mention there is a test case that test these error messages which i have altered to fit this schema.
If you are willing to make the changes, i would suggest to search for ` = 's' if len(` across the whole code base, and make the changes accordingly.
Right now i am AFK, texting you using the app.

Regards.